lib/portlayer/network/ipam.go

```diff
@@ -281,13 +283,25 @@ func (s *AddressSpace) ReserveIP4Range(firstIP net.IP, lastIP net.IP) (*AddressS
 		}
 
 		// found range
+		log.Infof("Reserving IP range [%s, %s]", firstIP.String(), lastIP.String())
 		s.reserveSubRange(firstIP, lastIP, i)
 		subSpace := NewAddressSpaceFromRange(firstIP, lastIP)
 		subSpace.Parent = s
 		return subSpace, nil
 	}
 
-	return nil, fmt.Errorf("could not find IP range")
+	var err error
+	if compareIP4(firstIP, s.Pool.FirstIP) > 0 && compareIP4(lastIP, s.Pool.LastIP) < 0 {
+		// IP range is within the pool but not found available
+		err = fmt.Errorf("Cannot reserve IP range [%s, %s].  Already in use", firstIP.String(), lastIP.String())
```

Comment:
error strings should not be capitalized, or contain sentence fragments. The following is much more concise (acronyms remain capitalized as its not contextual): 
```
IP range already in use: %s - %s
```

Reply:
This is what I had originally.  I want to keep symmetry with the release call, which explicitly states it's releasing.